src/certifier.mak

```diff
@@ -55,22 +55,42 @@ LINK=g++
 # Point this to the right place, if you have to, based on your machine's install:
 # PROTO=/usr/local/bin/protoc
 PROTO=protoc
+
+# protoc --python_out will generate this file
+PROTOC_PB2_PY = certifier_pb2.py
 AR=ar
 LL = ls -aFlrt
 
+# -----------------------------------------------------------------------------
 # Definitions needed for generating Python bindings using SWIG tool
 SWIG=swig
 
+# SWIG Debugging flags: Invoke as: $ SWIG_DEBUG=1 make ...
+#   -debug-tmsearch : Show typemap rules searched while processing interface
+#   -debug-tmused   : Show typemap rules finally applied
+#   -E              : Add this to display preprocessed output
+SWIG_DEBUG_FLAGS =
+ifdef SWIG_DEBUG
+SWIG_DEBUG_FLAGS = -debug-tmsearch -debug-tmused -E
+endif
```

Comment:
I'm just providing this make-hook to invoke as:

```
SWIG_DEBUG=1 make -f certifier.mak ...
```

So that SWIG built-in debugging flags can be turned ON.

These SWIG-debug flags are useful to understand and debug swig typemapping rules / code-flow.